protocol/messenger_handler.go

```diff
@@ -1702,7 +1702,7 @@ func (m *Messenger) HandleDeleteForMeMessage(state *ReceivedMessageState, delete
 		return errors.New("chat not found")
```

Comment:
hi @jrainville , could you pls explain more the TODO you add at line 1684 ? thank you

Reply:
generally , we prefer to add an issue link with TODO 🙂